repository/g.json

```diff
@@ -2334,6 +2334,16 @@
 					"branch": "master"
 				}
 			]
+		},
+		{
+			"name": "GNOME Theme",
+			"details": "https://github.com/mechtifs/sublime-gnome-theme",
+			"releases": [
```

Comment:
Your package is both a theme and a color scheme. It's probably a good idea to explain as much, and to add some labels.